queries.sql

```diff
@@ -0,0 +1,13 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered = TRUE;
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
- [ ] Hello, fantastic job on incorporating queries to address analytical questions! 😊 Upon reviewing the project guidelines, I observed that there's a specific query mentioned that hasn't been addressed in the current project. 🤔 Please double-check to ensure that all the questions outlined in the guidelines are covered by the queries. Thank you! 🚀

![Screenshot from 2023-11-20 19-36-56](https://github.com/axcell-0/vet-clinics/assets/112261237/072845b3-7055-4e0e-825c-4cfcbf93dbba)